en/docs/tutorials/construct-statements.md

```diff
@@ -0,0 +1,116 @@
+# Construct Statements
```

Comment:
Are we going to link to this tutorial via the statement editor documentation icon?
<img width="605" alt="statement editor guide" src="https://user-images.githubusercontent.com/9528121/187830711-d56bf319-f98e-4cca-a248-932bff13cbb8.png">
Was just wondering whether it makes sense to have a tutorial like this if the purpose is to provide reference info to guide users to build any kind of statement that they would want to construct.

Reply:
Added the initial steps for accessing Choreo for the version hosted from the Documentation site. Can have a separate version for the document linked from the UI without those steps.

Reply:
We discuss a separate location @sachiniSam can we share the design.

Reply:
Refer Frame 3 -> https://www.figma.com/file/Sj7NyIFdvvvVcEeGEeHTHt/Statement-Editor?node-id=7%3A2331